scripts/hid_configurator/configurator_cli.py

```diff
@@ -81,7 +82,11 @@ def perform_config(dev, args):
         else:
             print('Failed to fetch {}'.format(config_name))
     else:
-        config_value = int(args.value)
+        try:
+            config_value = value_type(args.value)
+        except:
```

Comment:
except must take an exception type to avoid catching more exceptions than intended.